src/safeds/data/image/containers/_image.py

```diff
@@ -434,6 +430,38 @@ def adjust_contrast(self, factor: float) -> Image:
         else:
             return Image(func2.adjust_contrast(self._image_tensor, factor * 1.0), device=self.device)
 
+    def adjust_color_balance(self, factor: float) -> Image:
+        """
+        Return a new `Image` with adjusted color balance.
+
+        The original image is not modified.
+
+        Parameters
+        ----------
+        factor: float
+            If factor > 1, increase color balance of image.
+            If factor = 1, no changes will be made.
+            If factor < 1, make image greyer.
+            Has to be bigger than or equal to 0.
```

Comment:
I'd suggest changing the order of those entries, so the intervals are listed from low to high:

```suggestion
            Has to be bigger than or equal to 0.
            If 0 <= factor < 1, make image greyer.
            If factor = 1, no changes will be made.
            If factor > 1, increase color balance of image.
```